src/components/dockerRegistry/Docker.tsx

```diff
@@ -1102,6 +1285,23 @@ const DockerForm = ({
         ) {
             return (
                 <>
+                    <div className="dc__position-rel dc__hover mb-20">
+                        <span className="form__input-header pb-20">
+                            How do you want Devtron to connect with this cluster?
+                        </span>
+                        <span className="pb-20">
+                            {RemoteConnectionRadio && (
+                                <RemoteConnectionRadio
+                                    connectionMethod={customState.remoteConnectionConfig.connectionMethod}
```

Comment:
can we send only `customState.remoteConnectionConfig` to the child extract there?

Reply:
we need it to make it generalized for both docker and clusters